sonaranalyzer-dotnet/rspec/cs/S3168_c#.html

```diff
@@ -46,4 +46,7 @@ <h2>Compliant Solution</h2>
   }
 }
 </pre>
+<h2>Exceptions</h2>
```

Comment:
Are all of these exceptions actually implemented?

Reply:
Yes, I've checked.